services/hasura/migrations/1677672007088_create_table_public_branches/up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE "public"."branches" ("id" uuid NOT NULL DEFAULT gen_random_uuid(), "created_at" timestamptz NOT NULL DEFAULT now(), "updated_at" timestamptz NOT NULL DEFAULT now(), "name" text NOT NULL DEFAULT 'main', "user_id" uuid NOT NULL, "datasource_id" uuid NOT NULL, PRIMARY KEY ("id") , FOREIGN KEY ("user_id") REFERENCES "public"."users"("id") ON UPDATE cascade ON DELETE cascade, FOREIGN KEY ("datasource_id") REFERENCES "public"."datasources"("id") ON UPDATE cascade ON DELETE cascade, UNIQUE ("name", "datasource_id"));
```

Comment:
maybe you need to delete previous migrations which creates and drops `branches` table via `hasura migrate status` and `hasura migrate delete`